src/riak_kv_vnode.erl

```diff
@@ -110,6 +110,21 @@
                 reqid :: term(),
                 target :: pid()}).
 
+-record(counter_state, {
```

Comment:
Minor nit: suggest renaming this record to epoch_counter (and giving the state record field that same name) to be more explicit